src/i18n/en.js

```diff
@@ -224,12 +222,12 @@ export default {
     error_checkbox_policy: `Ouch! You haven't accepted the Doppler's Privacy Policy.`,
     error_email_already_exists: `Ouch! You already have a Doppler account.`,
     error_invalid_domain_to_register_account: `Ouch! Invalid Email to create an account.`,
-    error_invalid_email_address: `Ouch! Invalid email address`,
+    error_invalid_email_address: `Ouch! Enter a valid Email.`,
     error_invalid_login: `Ouch! There is an error in your Username or Password. Please, try again.`,
-    error_password_character_length: `At least 8 characters`,
+    error_password_character_length: `8 characters minimum`,
     error_password_digit: `One number`,
     error_password_letter: `One letter`,
-    error_password_safe: `Your password is safe!`,
+    error_password_safe: `Your password is secure!`,
     error_phone_invalid: `Ouch! The phone number is not valid`,
```

Comment:
error_phone_invalid: `Ouch! Enter a valid phone number.`,

Reply:
@CarlosSampedro is right.